src/delta.rs

```diff
@@ -539,6 +622,54 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_diff_unified_two_files() {
+        let options = get_command_line_options();
+        let output = strip_ansi_codes(&run_delta(DIFF_UNIFIED_TWO_FILES, &options)).to_string();
+        let mut lines = output.split('\n');
+
+        // Header
+        assert_eq!(
+            lines.nth(1).unwrap(),
+            "comparing: one.rs\t2019-11-20 ⟶   src/two.rs\t2019-11-18"
```

Comment:
I'm hesitant about including these dates with the file names. What's your thinking on this? I'm kind of thinking that timestamps are orthogonal to the question of how two files differ and we should use 

```comparing: one.rs ⟶   src/two.rs```.